code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -420,7 +420,7 @@
 /datum/reagent/hydrogen_peroxide/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume)//Splashing people with h2o2 can burn them !
 	. = ..()
 	if(methods & TOUCH)
-		exposed_mob.adjustFireLoss(2, 0) // burns
+		exposed_mob.adjustFireLoss(2)
```

Comment:
?

Reply:
I don't know how that got in there.

Reply:
Oh it's the mirror in https://github.com/Skyrat-SS13/Skyrat-tg/pull/20518